internal/cmddiff/cmddiff.go

```diff
@@ -29,7 +29,7 @@ import (
 func NewRunner(parent string) *Runner {
```

Comment:
Where's the logic for building the absolute path to package from the PKG_PATH argument?
This logic needs to be extracted into a common utility and shared with all commands.
Typically, we should also pass around specific types such i.e.`UniquePath` instead of string:

https://github.com/GoogleContainerTools/kpt/blob/next/internal/pkg/pkg.go#L26

Reply:
I see that most of the commands don't need to standardize the path to absolute path. I will do it as a separate story for standardizing it for all the commands. This Pr is only for pkg commands.

Reply:
Will this be in a future PR? Can you make sure all comment thread are resolved.

Reply:
I will do it as part of this PR. Moved PR to WIP.